eZ/Publish/Core/Base/Exceptions/ContentFieldValidationException.php

```diff
@@ -53,4 +64,27 @@ public function getFieldErrors()
     {
         return $this->errors;
     }
+
+    private function generateValidationErrorsMessages(): string
+    {
+        $message = '';
+        foreach ($this->getFieldErrors() as $validationErrors) {
+            foreach ($validationErrors as $validationError) {
+                if (is_array($validationError)) {
+                    foreach ($validationError as $item) {
+                        $message .= $this->generateValidationErrorMessage($item);
+                    }
+                } else {
+                    $message .= $this->generateValidationErrorMessage($validationError);
+                }
+            }
+        }
+
+        return $message;
+    }
+
+    private function generateValidationErrorMessage(ValidationError $validationError): string
+    {
+        return sprintf("\n- %s", $validationError->getTranslatableMessage());
+    }
```

Comment:
This should rather be an array which gets to be `implode` after collecting. Seems performance wise better.